dandi/consts.py

```diff
@@ -100,6 +112,14 @@
 
 file_operation_modes = ["dry", "simulate", "copy", "move", "hardlink", "symlink"]
 
+#
+# Download (upload?) specific constants
+#
+# Chunk size when iterating a download (and upload) body. Taken from girder-cli
+# TODO: should we make them smaller for download than for upload?
+# ATM used only in download
+MAX_CHUNK_SIZE = os.environ.get("DANDI_MAX_CHUNK_SIZE", 1024 * 1024 * 8)  # 64
```

Comment:
You need to cast this to an `int`, as the value from `os.environ` is a string.

Reply:
d'oh, right,THANKS! done in 09b219660e77f88b581959447a5eee8af0f7facf